src/main/scala/centaur/test/Test.scala

```diff
@@ -67,7 +66,8 @@ object Operations {
         // Collect only the parameters which exist:
         val params = List("wdlSource" -> Option(workflow.data.wdl),
           "workflowInputs" -> workflow.data.inputs,
-          "workflowOptions" -> workflow.data.options) collect { case (name, Some(value)) => (name, value) }
+          "workflowOptions" -> WorkflowOptions(workflow.data.options).insertSecrets.options
```

Comment:
It looks `WorkflowOptions` is used on one line only to box the options, invoke `insertSecrets`, and unbox.  Perhaps it would be more straightforward to have a function that just operates directly on the options?